modules/openapi-generator/src/main/resources/r/api.mustache

```diff
@@ -206,8 +206,21 @@
 
       {{/requiredParams}}
       {{#allParams}}
+      {{^isNullable}}
+      if (!missing(`{{paramName}}`) && is.null(`{{paramName}}`)) {
+        {{#useDefaultExceptionHandling}}
+        stop("Invalid value for `{{paramName}}` when calling {{classname}}${{operationId}}, `{{paramName}}` is not nullable")
+        {{/useDefaultExceptionHandling}}
+        {{#useRlangExceptionHandling}}
+        rlang::abort(message = "Invalid value for `{{paramName}}` when calling {{classname}}${{operationId}}, `{{paramName}}` is not nullable",
+                     .subclass = "ApiException",
+                      ApiException = ApiException$new(status = 0,
+                      reason = "Invalid value for `{{paramName}}` when calling {{classname}}${{operationId}}, `{{paramName}}` is not nullable"))
+        {{/useRlangExceptionHandling}}
+      }
+      {{/isNullable}}
```

Comment:
thanks for the pr

2 questions:

1. looks like required parameters are processed again for null check. shall we update the block in line 195 instead?

2. since this is in the `allParams` loop/list, that means optional parameters (if defined) will be processed as well. would optional parameters work in this new code block (null check)?